content/post/119-how-we-built-this.md

```diff
@@ -0,0 +1,60 @@
+---
+date: 2021-01-21
+url: /2021-01-21-how-we-built-this/
+title: Integrating IPFS and Brave – How We Built This
+author: Mitch Wagner
+tags: browsers
+header_image: how-we-built-this.png
+snippet: This integration marks a major milestone for IPFS, and lays the groundwork for further experiments into improving the experience of interacting with the network via web browsers.
+---
+
+<p style="text-align: center;">
+  <img src="/img/how-we-built-this.png" alt="Integrating IPFS and Brave – How We Built This">
+</p>
+
+You may have heard the news: as of [release 1.19.86](https://github.com/brave/brave-browser/releases/tag/v1.19.86), the Brave browser has officially added [InterPlanetary File System (IPFS)](https://blog.ipfs.io/2021-01-19-ipfs-in-brave/) support! This work is the result of a multi-year effort to bring the two projects together. In this post, we discuss the process in bringing this collaboration to fruition, and take a look under the hood to see how we accomplished this integration!
+
+## Collaboration
+Both Brave and IPFS have a long history of successful partnerships with other projects and vendors. Brave now ships with built-in wallets for cryptocurrencies, private windows that leverage Tor, and a highly integrated VPN option. IPFS, meanwhile, has collaborated with such names as Microsoft (developing a [decentralized identity stack](https://blog.ipfs.io/2020-06-11-identity-ipfs-ion/)), Netflix (experimenting [fetching Docker images via IPFS](https://blog.ipfs.io/2020-02-14-improved-bitswap-for-container-distribution/)), and NixOS (decentralizing [source code and build products](https://blog.ipfs.io/2020-09-08-nix-ipfs-milestone-1/)).
+
+![](119-brave-how-we-built-this/ipfs-early.png)
```

Comment:
Needs fixing (does not load from gateway)

Reply:
Are you commenting on the screenshots?